bin/tictactoe

```diff
@@ -1,3 +1,33 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
```

Comment:
@Juanriv45 for lines 4-33 it would be great to abstract this into a CLI class within our lib folder. keep the bin file simple

something like 

```CLI.new.start_game```